config/examples/stm32c0.config

```diff
@@ -0,0 +1,34 @@
+ARCH?=ARM
+TARGET?=stm32c0
+SIGN?=ED25519
+#Using SHA384 with ED25519 saves about 2kB without SHA256 code
+HASH?=SHA384
+DEBUG?=0
+VTOR?=1
+CORTEX_M0?=1
+NO_ASM?=0
+EXT_FLASH?=0
+SPI_FLASH?=0
+ALLOW_DOWNGRADE?=0
+NVM_FLASH_WRITEONCE?=1
+WOLFBOOT_VERSION?=0
+V?=0
+SPMATH?=1
+RAM_CODE?=1
```

Comment:
Note: With `RAM_CODE?=0` the code size is about 670 bytes smaller. However that prevents wolfBoot from having the ability to self-update. Also its about 75% slower. Recommend the default stays enabled, but might be worth documenting.

Reply:
OTOH immutable wolfboot if that smaller would allow for larger partition and a better geometry (10K wolfBoot, 10K partition size, BOOT=0x08002800 UPDATE=0x08005000 SWAP=0x08007800 with 6KB max effective application size and 2 sectors reserved for NVM_FLASH_WRITEONCE in each partition)

Reply:
Is the speed impact due to the code running out of the flash?  I thought this had an instruction cache (small), but no data cache.  I'll extend the documentation to provide those tradeoffs.  Is the ONLY thing I need to do to disable wolfBoot updates is set RAM_CODE=0?  Seems like we should be ok with pushing a bunch of the code into RAM during boot since the ram will be recovered during the app boot.